utils/jira_functions.py

```
@@ -0,0 +1,180 @@
+import json
+import os
+from urllib import response
+import requests
+import sys
+from collections import defaultdict
+from urllib3.util import Retry
+
+import requests
```

Ruff (0.14.14)

[error] Redefinition of unused `requests` from line 4: `requests` redefined here

Remove definition: `requests`

(F811)